spec/validators/valid_for_notify_validator_spec.rb

```diff
@@ -71,7 +71,6 @@
         'brackets(in)local@domain.com',
         'infö@german-financial-services.de',
         'info@german-financial-services.vermögensberatung',
-        'japanese-info@例え.テスト',
```

Comment:
Why is this valid but umlauts, grave and accent are not?

Is there documentation provided by the service we can link to?

Reply:
> Why is this valid but umlauts, grave and accent are not?
Tbh, we (the team) need to talk about email formats as there are more international characters that we aren't handling atm.

> Is there documentation provided by the service we can link to?
Sure, https://www.notifications.service.gov.uk/using-notify

Reply:
Can we put this link in the documentation you created?